deploy/olm-catalog/pulp-operator/manifests/pulp-operator.clusterserviceversion.yaml

```diff
@@ -130,32 +130,32 @@ spec:
         - urn:alm:descriptor:com.tectonic.ui:advanced
         - urn:alm:descriptor:io.kubernetes:Secret
         - urn:alm:descriptor:com.tectonic.ui:fieldDependency:ingress_type:Route
-      - displayName: Pulp Storage Type
+      - displayName: Storage type
         path: storage_type
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:select:File
         - urn:alm:descriptor:com.tectonic.ui:select:S3
         - urn:alm:descriptor:com.tectonic.ui:select:Azure
-      - displayName: Pulp File Storage Configuration
+      - displayName: File storage configuration
         path: file_storage
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:fieldDependency:storage_type:File
-      - displayName: Pulp File Storage Configuration
+      - displayName: File storage access mode
         path: file_storage.access_mode
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:fieldDependency:storage_type:File
         - urn:alm:descriptor:com.tectonic.ui:select:ReadWriteMany
-      - displayName: Pulp S3 Storage Secret
+      - displayName: S3 storage secret
         path: object_storage_s3_secret
         x-descriptors:
         - urn:alm:descriptor:io.kubernetes:Secret
         - urn:alm:descriptor:com.tectonic.ui:fieldDependency:storage_type:S3
-      - displayName: Pulp Azure Storage Secret
+      - displayName: Azure storage secret
         path: object_storage_azure_secret
         x-descriptors:
         - urn:alm:descriptor:io.kubernetes:Secret
         - urn:alm:descriptor:com.tectonic.ui:fieldDependency:storage_type:Azure
-      - displayName: Pulp Settings
+      - displayName: Pulp settings
```

Comment:
Was it intentional to keep `Pulp` here?

Reply:
It's currently a hidden value so I kept it since its very advance usage. Having it here provides some linkage to the documents a user would need.